core/src/apps/cosmos/networks.py

```
@@ -1,9 +1,25 @@
-from typing import Iterator
+from typing import TYPE_CHECKING
 
 from trezor.strings import format_amount
 
 from . import ICON, PRIMARY_COLOR
 
+if TYPE_CHECKING:
+    from typing import Iterator
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)